src/test/rustdoc-ui/intra-doc/disambiguator-mismatch.stderr

```diff
@@ -2,94 +2,138 @@ error: incompatible link kind for `S`
   --> $DIR/disambiguator-mismatch.rs:14:14
    |
 LL | /// Link to [struct@S]
-   |              ^^^^^^^^ help: to link to the enum, prefix with `enum@`: `enum@S`
+   |              ^^^^^^^^
```

Comment:
I notice that this file has no test where _multiple_ alternatives should be suggested, could we add one?

Reply:
Sure!

Reply:
We actually can't have two items with in the same name in a module. Did you have something else in mind maybe?

Reply:
@GuillaumeGomez there could be two items with the same name and different namespaces:
```
const C: usize = 0;
macro_rules! C { () => {}; }
fn C() {}
```

Reply:
Sadly, it's not handled yet apparently... :'(

Reply:
Can you add a test so we can see for ourselves? It's ok that it's not handled, but we should at least know how it behaves.

Reply:
I did: https://github.com/rust-lang/rust/pull/87285/files#diff-24d855006c44e30e19269dd00e9f07f72f50c5d0e97ceaacbc2f6fafe79eb2caR4